lib/velum/dex/oidc.rb

```diff
@@ -0,0 +1,25 @@
+require "base64"
+
+module Velum
+  # This class offers the integration between ruby and the Saltstack API.
+  module Dex
+    class << self
+      def oidc_connectors_as_pillar
+        oidc_connectors = DexConnectorOidc.all.map do |con|
+          {
+            "type"          => "oidc",
+            "id"            => "oidc-" + con.id.to_s,
+            "name"          => con.name,
+
+            "provider_url"  => con.provider_url,
+            "client_id"     => con.client_id,
+            "client_secret" => con.client_secret,
+            "callback_url"  => con.callback_url,
+            "basic_auth"    => con.basic_auth
+          }
+        end
+        oidc_connectors
```

Comment:
you can just omit the variable decalaration, as the map will be returned implicitly

```ruby
      def oidc_connectors_as_pillar
        DexConnectorOidc.all.map do |con|
          {
            "type"          => "oidc",
            "id"            => "oidc-" + con.id.to_s,
            "name"          => con.name,

            "provider_url"  => con.provider_url,
            "client_id"     => con.client_id,
            "client_secret" => con.client_secret,
            "callback_url"  => con.callback_url,
            "basic_auth"    => con.basic_auth
          }
        end
      end
```

Reply:
Whoops.  Fixed in 1240800. :)

Reply:
And now Rubocop rightfully complains about the useless assignment.  Whoops, again.  I'll fix that when I push up the fix for the alerts in the other conversation. :)

Reply:
Actually fixed in 539a210